command/status.go

```diff
@@ -134,3 +138,15 @@ func (c *StatusCommand) Run(args []string) int {
 
 	return cmd.Run(argsCopy)
 }
+
+func (c *StatusCommand) outputMultipleMatches(id string, matches map[contexts.Context][]string) {
```

Comment:
Maybe "logErrorForMultipleMatches", or a function name that is a bit more descriptive.